nilearn/datasets/atlas.py

```diff
@@ -1059,19 +1167,23 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
 
     Parameters
     ----------
-    ordered_regions : bool, optional
+    ordered_regions : :obj:`bool`, optional
         ROIs from same networks are grouped together and ordered with respect
         to their names and their locations (anterior to posterior).
         Default=True.
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, contains:
 
-        - "rois": coordinates of 160 ROIs in MNI space
-        - "labels": ROIs labels
-        - "networks": networks names
+        - 'rois': :class:`numpy.recarray`, rec array with the coordinates
+          of the 160 ROIs in :term:`MNI` space.
+        - 'labels': :class:`numpy.ndarray` of :obj:`str`, list of label
```

Comment:
Based on other atlases, it seems like this should be a list. Maybe something worth opening a separate issue for?